[2025-04-10T21:50:23.417+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: generate_and_verify_data.run_generate_script manual__2025-04-10T21:48:43.410592+00:00 [queued]>
[2025-04-10T21:50:23.428+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: generate_and_verify_data.run_generate_script manual__2025-04-10T21:48:43.410592+00:00 [queued]>
[2025-04-10T21:50:23.429+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-04-10T21:50:23.447+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): run_generate_script> on 2025-04-10 21:48:43.410592+00:00
[2025-04-10T21:50:23.455+0000] {standard_task_runner.py:57} INFO - Started process 12631 to run task
[2025-04-10T21:50:23.459+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'generate_and_verify_data', 'run_generate_script', 'manual__2025-04-10T21:48:43.410592+00:00', '--job-id', '1031', '--raw', '--subdir', 'DAGS_FOLDER/dags_1.py', '--cfg-path', '/tmp/tmpt6g7954x']
[2025-04-10T21:50:23.463+0000] {standard_task_runner.py:85} INFO - Job 1031: Subtask run_generate_script
[2025-04-10T21:50:23.518+0000] {task_command.py:415} INFO - Running <TaskInstance: generate_and_verify_data.run_generate_script manual__2025-04-10T21:48:43.410592+00:00 [running]> on host 7889a671eaba
[2025-04-10T21:50:23.614+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='generate_and_verify_data' AIRFLOW_CTX_TASK_ID='run_generate_script' AIRFLOW_CTX_EXECUTION_DATE='2025-04-10T21:48:43.410592+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-10T21:48:43.410592+00:00'
[2025-04-10T21:50:23.616+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-10T21:50:23.617+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python3 /opt/***/data/generate_script.py && echo "Script executed successfully." || echo "Script failed."']
[2025-04-10T21:50:23.631+0000] {subprocess.py:86} INFO - Output:
[2025-04-10T21:50:24.781+0000] {subprocess.py:93} INFO - Script executed successfully.
[2025-04-10T21:50:24.782+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-10T21:50:24.825+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=generate_and_verify_data, task_id=run_generate_script, execution_date=20250410T214843, start_date=20250410T215023, end_date=20250410T215024
[2025-04-10T21:50:24.894+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-10T21:50:24.929+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
